js/src/containers/Facebook.js

```diff
@@ -0,0 +1,122 @@
+/* External dependencies */
```

Comment:
I've added comments to `Twitter.js`. As this one is a sort of copy, most of my points there also go for this file.